src/dev/ci_setup/load_env_keys.sh

```diff
@@ -34,6 +34,9 @@ else
   PERCY_TOKEN=$(retry 5 vault read -field=value secret/kibana-issues/dev/percy)
   export PERCY_TOKEN
 
+  FOSSA_API_KEY=$(vault read -field=token secret/jenkins-ci/fossa/api-token)
```

Comment:
please include the `retry 5` here, vault can be a little finicky and needs to be retried